### PR TITLE
Adding new MF-MF causal relations

### PR DESCRIPTION
**Pull request by vanaukenk**
Note that these relations have been added to RO, but there isn't yet an official RO release including them (latest RO release is 2022-09-07).

### CODE REVIEW

**Comment by vanaukenk** [comment]
I think we're still waiting on a new build of go-lego-reacto in order to validate the new relations.  It looks like the last successful snapshot was 2022-10-24?
If so, that would explain the continued failure, as the RO was released on 2022-10-26.

**Comment by vanaukenk** [comment]
2022-10-28 snapshot contains the new RO relations in go-lego-reacto.
Still not certain why the Travis checks are failing.

**Comment by balhoff** [comment]
@vanaukenk the tests were broken, but I fixed them. I merged the master branch into this one to make them run.

**Comment by vanaukenk** [comment]
Thank you so much @balhoff !

File: shapes/go-cam-shapes.shex [hunk 1]
```diff
@@ -93,6 +93,11 @@ PREFIX positively_regulates: <http://purl.obolibrary.org/obo/RO_0002213>
 PREFIX directly_regulates: <http://purl.obolibrary.org/obo/RO_0002578>
 PREFIX directly_negatively_regulates: <http://purl.obolibrary.org/obo/RO_0002630>
 PREFIX directly_activates: <http://purl.obolibrary.org/obo/RO_0002406>
+PREFIX indirectly_regulates: http://purl.obolibrary.org/obo/RO_0012012
+PREFIX indirectly_positively_regulates: http://purl.obolibrary.org/obo/RO_0002407
+PREFIX indirectly_negatively_regulates: http://purl.obolibrary.org/obo/RO_0002409
+PREFIX constitutively_upstream_of: http://purl.obolibrary.org/obo/RO_0012009
+PREFIX removes_input_for: http://purl.obolibrary.org/obo/RO_0012010
```
Comment (balhoff) [inline shapes/go-cam-shapes.shex:100]: @vanaukenk I figured it out — you need angle brackets around these IRIs. :-)

**Review by balhoff — CHANGES_REQUESTED** [review]

**Comment by vanaukenk** [comment]
Ugh.  Thanks @balhoff

**Pull request merged**